docker-compose.override.yml

```diff
@@ -22,6 +22,13 @@ services:
     ports:
       - "3030:8080"
 
+  sso:
+    environment:
+      KEYCLOAK_USER: admin
+      KEYCLOAK_PASSWORD: Pa55w0rd
```

Comment:
Antar passord her er bare for test ? Skal dette inn i github på permanent basis må vi bruke environment variabler.

Reply:
Dette blir jo passord kun på localhost. For de andre miljøene legger jeg dem inn i konfigfil i Jenkins og Helm. Men vi har såvidt jeg vet ingen andre passord i docker-compose.override.yml. Det ryddigste er å legge dem i environment-variable:
environment:
  KEYCLOAK_USER=${KEYCLOAK_USER}
  KEYCLOAK_PASSWORD=${KEYCLOAK_PASSWORD}

Reply:
Jeg lagret nå .envrc fil som skal ikke sjekkes inni git. Der ligger secret til idporten client. Det kan vi ikke ha i git.

Jeg kan håndtere dev admin bruker samme møte

Reply:
høres ut som en bra løsning